docs/data/charts/bars/bars.md

```diff
@@ -159,14 +159,10 @@ If you're using composition, you can get those click event as follows.
 Notice that the `onAxisClick` will handle both bar and line series if you mix them.
 
 ```jsx
-import ChartsOnAxisClickHandler from '@mui/x-charts/ChartsOnAxisClickHandler';
-// ...
-
-<ChartContainer>
+<ChartContainer onAxisClick={onAxisClick}>
   {/* ... */}
-  <ChartsOnAxisClickHandler onAxisClick={onAxisClick} />
   <BarPlot onItemClick={onItemClick} />
-</ChartContainer>;
+</ChartContainer>
```

Comment:
Has this composition section become obsolete in both bar and line? I believe the idea was to show that to add interaction you need to use `<ChartsOnAxisClickHandler />` in your composition, but that is not the case anymore.

Also should we create a codemod for this `ChartsOnAxisClickHandler` removal? 🤔

Reply:
> Has this composition section become obsolete

I thaught first to remove it, but it's still relevant to show that the `onAxisClick` needs to be pass to the container and `onItemClick` to the plot element